test/apps/widgets-9.2/app.ts

```diff
@@ -0,0 +1,152 @@
+  _StatsWidget
+} from '@deck.gl/widgets';
+import '@deck.gl/widgets/stylesheet.css';
+
+// source: Natural Earth http://www.naturalearthdata.com/ via geojson.xyz
+const COUNTRIES =
+  'https://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_50m_admin_0_scale_rank.geojson'; //eslint-disable-line
+const AIR_PORTS =
+  'https://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_10m_airports.geojson';
+
+const INITIAL_VIEW_STATE = {
+  latitude: 51.47,
+  longitude: 0.45,
+  zoom: 4,
+  bearing: 0,
+  pitch: 30
+};
+
+const deck = new Deck({
+  initialViewState: INITIAL_VIEW_STATE,
+  controller: true,
+  layers: [
+    new WMSLayer({
+      data: 'https://ows.terrestris.de/osm/service',
+      serviceType: 'wms',
+      layers: ['OSM-WMS']
+    }),
+    new GeoJsonLayer({
+      id: 'base-map',
+      data: COUNTRIES,
+      // Styles
+      stroked: true,
+      filled: true,
+      lineWidthMinPixels: 2,
+      opacity: 0.4,
+      getLineColor: [60, 60, 60],
+      getFillColor: [200, 200, 200]
+    }),
+    new GeoJsonLayer({
+      id: 'airports',
+      data: AIR_PORTS,
+      // Styles
+      filled: true,
+      pointRadiusMinPixels: 2,
+      pointRadiusScale: 2000,
+      getPointRadius: f => 11 - f.properties.scalerank,
+      getFillColor: [200, 0, 80, 180],
+      // Interactive props
+      pickable: true,
+      autoHighlight: true
+    }),
+    new ArcLayer({
+      id: 'arcs',
+      data: AIR_PORTS,
+      dataTransform: d => d.features.filter(f => f.properties.scalerank < 4),
+      // Styles
+      getSourcePosition: f => [-0.4531566, 51.4709959], // London
+      getTargetPosition: f => f.geometry.coordinates,
+      getSourceColor: [0, 128, 200],
+      getTargetColor: [200, 0, 80],
+      getWidth: 1
+    })
+  ],
+  widgets: [
+    new _GeocoderWidget(),
+    new ZoomWidget(),
+    new CompassWidget(),
+    new FullscreenWidget(),
+    new ScreenshotWidget(),
+    new ResetViewWidget(),
+    new _FpsWidget(),
+    new _LoadingWidget(),
+    new _ScaleWidget({placement: 'bottom-right'}),
+    new _ThemeWidget(),
+    new _ContextMenuWidget({
+      getMenuItems: (info: PickingInfo) => {
+        const name = info.layer?.id === 'airports' && info.object?.properties.name;
+        console.log('Context menu:', name);
+        return (
+          name && [
+            {key: 'airport', label: `${name}`},
+            {key: 'open', label: 'Open in new tab'},
+            {key: 'favorite', label: 'Set as favorite'},
+            {key: 'filter', label: 'Exclude from filter'}
+          ]
+        );
+      }
+    }),
+    new _InfoWidget({mode: 'hover', getTooltip}),
+    new _InfoWidget({mode: 'click', getTooltip}),
+    new _TimelineWidget({
+      placement: 'bottom-left',
+      timeRange: [0, 24],
+      step: 1,
+      initialTime: 0,
+      playInterval: 1000,
+      // eslint-disable-next-line no-console, no-undef
+      onTimeChange: time => console.log('Time:', time)
+    }),
+    new _ViewSelectorWidget(),
```

Comment:
Not really sure what this widget should be doing, it feels broken


https://github.com/user-attachments/assets/11be242c-3c69-43c0-a530-50181f182798

Reply:
https://github.com/user-attachments/assets/d62ff27a-2540-4549-a56a-7fa35d29b7fa

Changed it to reuse the same components ZoomWidget uses, so not "IconMenu" is effectively a popup button group. Also, I changed it so the inline SVG's stroke value uses the CSS icon color variable provided by the theme. It just doesn't have the same "hover" color polish as the other icons since inline CSS doesn't natively support pseudo classes